handbook/engineering/web/goals.md

```diff
@@ -4,29 +4,24 @@
 
 ### Long term
 
-**_Deliver the full, unique value of [extensions](https://docs.sourcegraph.com/extensions) to our users._**
+**_The web team has three areas of focus: creating and maintaining a highly usable and intentionally designed webapp interface; delivering the full, unique value of [extensions](https://docs.sourcegraph.com/extensions) to our users; and developing code insights into an entirely new featureset._**
 
-**Outcome**: Our webapp, browser extensions and native integrations are great platforms to provide our users unique value through extensions. These platforms are well-maintained, consistent, easy to setup, well-documented, well-tested, performant, and show their power through convincing extensions built on top of them. The extensions platform and API provide powerful capabilities to extension developers and a great developer experience.
+**Outcome**: Our webapp, browser extensions and native integrations, and code insights all provide our users unique value. These platforms are well-maintained, well-designed, consistent, easy to setup, well-documented, well-tested, performant, and leave users impressed. The webapp helps users discover and employ the full power of Sourcegraph effectively, with a high quality and highly usable interface. The extensions and extensions API provide powerful capabilities to users and a great experience for extension developers. Code insights expose the value of code metadata to users at all levels of an organization. 
 
 ### Medium term
 
 To reach our long-term goal, we set the following medium-term goals to guide our short-term iteration plans.
-We will tackle these medium-term goals in order, though expect to have some work done in parallel as we progress.
 
-1. **Make the products that extensions are build on (web app, code host integrations) more consistent and improve discoverability.**
+1. **Make the Sourcegraph web interfaces more consistent and improve discoverability of Sourcegraph features.**
```

Comment:
**suggestion (non-blocking)**: Can we consider converting this section into a nested numbered list so it's easy to refer to a goal/sub-goal

1. Goal
    1. Sub-goal
    1. Sub-goal